### PR TITLE
LPS-154327 [Raylife AP] Create Dashboard Recent Applications Remote App

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/RecentApplications.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+	const handleAddApplication = () => {
+		alert('Add Application Action');
+	};
+
+	const handleDeleteApplication = (externalReferenceCode: string) => {
+		deleteApplicationByExternalReferenceCode(externalReferenceCode);
+
+		const newApplications = applications.filter(
+			(item) => !(item.key === externalReferenceCode)
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/RecentApplications.tsx:80]: Please, change ```item``` to ```application```.

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :fire: But needs some adjustments.

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
Hey @silasbsi , do not forward this. I was looking at the portal and the layout is working strangely.

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by JoMaAlves** [comment]
ci:forward

**Pull request closed without merging**